content/apps/managing-teammates.md

```diff
@@ -5,32 +5,41 @@ menu:
 title: Managing Teammates
 ---
 
-To add a new teammate to your organization:
+## Invite a teammate
+
+If you're part of an agency team using cloud.gov, you can invite other people at your agency to get cloud.gov accounts:
```

Comment:
Are people allowed to invite people who aren't working for the federal government in some way (employees or contractors)?